src/cli/cli_srp_client.cpp

```diff
@@ -288,14 +514,67 @@ template <> otError SrpClient::Process<Cmd("service")>(Arg aArgs[])
     otError error = OT_ERROR_NONE;
     bool    isRemove;
 
+    /**
+     * @cli srp client service
+     * @code
+     * srp client service
+     * instance:"ins2", name:"_test2._udp,_sub1,_sub2", state:Registered, port:111, priority:1, weight:1
+     * instance:"ins1", name:"_test1._udp", state:Registered, port:777, priority:0, weight:0
+     * Done
+     * @endcode
+     * @par api_copy
+     * #otSrpClientGetServices
+     */
     if (aArgs[0].IsEmpty())
     {
         OutputServiceList(0, otSrpClientGetServices(GetInstancePtr()));
     }
+    /**
+     * @cli srp client service add 
+     * @code
+     * srp client service add ins1 _test1._udp 777
+     * Done
+     * @endcode
+     * @cparam srp client service add @ca{instancename} @ca{servicename} @ca{port} [@ca{priority}] [@ca{weight}] [@ca{txt}]
+     * Use names for the `instancename` and `servicename` parameters that help
+     * describe the behavior and function that the device provides. The
+     * 'servicename` paramater can can optionally include a list of service
+     * subtype labels that are separated by commas. The examples here use
+     * generic naming.
+     * The `priority` and `weight` (`weight` is given as a `uint16_t` value)
+     * parameters are optional, and if not provided
+     * zero is used. The optional `txt` parameter allows for additional information
+     * to be added to describe the service. The `txt` vaule must be in
+     * hex-string format and is treated as an already encoded TXT data byte
+     * sequence. If not provided, `txt` is considered empty.
```

Comment:
I recommend removing the "If not provided, `txt` is considered empty.".

The default case is an array containing just single `0x00` byte (indicating no content).

Reply:
@abtink - Thanks Abtin. Do you want me to add the info about the default case?